### PR TITLE
Attempt to fix simple problems with backend.go

### PR DESCRIPTION
**Pull request by brian-peter-dickson**
Please review, haven't tested or even tried to compile...

### CODE REVIEW

File: backend/backend.go [hunk 1]
```diff
@@ -78,16 +126,16 @@ func (backend *Backend) ProcessCENReport(cenReport *CENReport) (err error) {
 	// store the cenreportID in cenkeys table, one row per key
 	for _, cenKey := range cenKeys {
 		cenKey := strings.Trim(cenKey, " \n")
-		if len(cenKey) > 30 && len(cenKey) <= 32 {
-			_, err = stmtKeys.Exec(cenKey, reportID, curTS)
```
Comment (bleggett) [inline backend/backend.go:82]: Could we get a new unit test in `backend_test.go` that demonstrates that the timestamp used is in fact the one coming from the report?

Comment (brian-peter-dickson) [reply backend/backend.go:82]: Fixed in main code doing the generation of sample data...

File: backend/backend.go [hunk 2]
```diff
@@ -78,16 +126,16 @@ func (backend *Backend) ProcessCENReport(cenReport *CENReport) (err error) {
...
+		if len(cenKey) > 62 && len(cenKey) <= 64 {
+			_, err = stmtKeys.Exec(cenKey, reportID, cenReport.ReportTimestamp)
```
Comment (bleggett) [inline backend/backend.go:130]: `ReportTimeStamp`

**Comment by bleggett** [comment]
FYI this doesn't compile as-is.

**Pull request merged**